### ISSUE TITLE
Can't do anything after `now logout`

### ISSUE THREAD

**Issue by kt3k** [vercel/vercel#3786]
After logged out from cli, both login and logout don't work (now v17.0.3)

```
$ now logout
Now CLI 17.0.3
Error! The content of "~/.now/auth.json" is invalid. No `token` property found inside. Run `now login` to authorize.
$ now login
Now CLI 17.0.3
Error! An unexpected error occurred!
  Error: The specified token is not valid
```

I even can't see the help message
```
$ now --help
Now CLI 17.0.3
Error! An unexpected error occurred!
  Error: The specified token is not valid
```

**Comment by kt3k** [comment]
I downgraded now to v16.x and I was able to login. After that I upgraded to v17 again and now I can use the cli v17 with logged-in state. Maybe this is a workaround, but it seems very inconvenient.

**Comment by devalnor** [comment]
Same issue here. I roll back to 16.7.3 and it's work.

**Comment by styfle** [comment]
Thanks for reporting this issue! It will be fixed in PR #3907.

**Comment by styfle** [comment]
You can try this out on the canary channel `npm i -g now@canary`.
